src/definitions/constants.rs

```diff
@@ -31,4 +31,11 @@ lazy_static! {
     // Value generated from get_selector_from_name('__validate_declare__')
     pub static ref VALIDATE_DECLARE_ENTRY_POINT_NAME: Felt =
         felt_str!("1148189391774113786911959041662034419554430000171893651982484995704491697075");
+    pub static ref TEST_SEQUENCER_ADDRESS: Felt =
+        felt_str!("4096");
+    pub static ref TEST_ERC20_CONTRACT_ADDRESS: Felt =
+        felt_str!("4097");
```

Comment:
This is part of the test, and does not apply to the real world, therefore it'd probably be better left in `internals.rs`.